static/src/stylesheets/module/_adslot.scss

```diff
@@ -165,6 +165,12 @@
     }
 }
 
+// Compatibility fix for top banner ads in IE11/Edge. Without this top banner
+// appears further down the page, and covers content, in immersive galleries.
+.immersive-header-container .top-banner-ad-container {
```

Comment:
Would it do any harm to add the `top` to the top banner everywhere, not only in immersive galleries? Let's try to keep the layout consistent.